tests/hardware_reboot/testdata/hardware_usb_reboot.txt

```diff
@@ -0,0 +1,83 @@
+# Simple test of USB passthrough functionality after reboot of guest
+
+{{$usb_dev := "2-2"}}
```

Comment:
Does this work with Xen?

Reply:
Good point. I see it on another address inside.
```
root@0ce327c6-7f9e-433d-b73f-9b7e5a096cdd:~# cat /sys/bus/usb/devices/1-1/product 
QEMU USB HARDDRIVE
```